cluster/gce/gci/configure-helper.sh

```diff
@@ -2438,6 +2438,9 @@ EOF
   if [[ "${ENABLE_DEFAULT_STORAGE_CLASS:-}" == "true" ]]; then
     setup-addon-manifests "addons" "storage-class/gce"
   fi
+  if [[ "${ENABLE_WINDOWS_STORAGE_CLASS:-}" == "true" ]]; then
+    setup-addon-manifests "addons" "storage-class/gce/windows"
```

Comment:
the actual file seems to be missing from the commit

Reply:
fixed it.